flask_appbuilder/security/forms.py

```diff
@@ -5,6 +5,7 @@
 
 from ..fieldwidgets import BS3PasswordFieldWidget, BS3TextFieldWidget
 from ..forms import DynamicForm
+from ..validators import PasswordComplexityValidator
```

Comment:
Just an opinion, but while we're on the topic of improving typing/readability, I somehow always find fully qualified imports more pleasant to read:

```python
from flask_appbuilder.validators import PasswordComplexityValidator
```
over
```python
from ..validators import PasswordComplexityValidator
```

Reply:
Currently I prefer them also, but I'm following the current code base "convention", I plan to change them on a different PR

Reply:
Agree. Relative imports are evil.

Reply:
Coming as strong advise in Airflow 2.2: https://github.com/apache/airflow/blob/main/docs/apache-airflow/modules_management.rst#dont-use-relative-imports